### DISCUSSION TITLE
Create a discord channel for MakeContributions Organization

### DISCUSSION THREAD

**Discussion by UG-SEP** [MakeContributions/DSA#384] [Ideas]
I am thinking is it better if we create a Discord for MakeContribution So people can collaborate with each other ask questions, send useful resources, general talk, etc it will be an easy way to interact with each other...
What do you all think @ming-tsai, @Arsenic-ATG, @aayushjain7, @Ashborn-SM

**Comment by Ashborn-SM** [comment]
@UG-SEP that's actually a good idea.

**Comment by ming-tsai** [comment]
Ok, I will create it

**Comment by ming-tsai** [comment]
Here we go the [Discord link](https://discord.gg/7t4BXVJC)

**Reply by chrismvelez97** [reply]
Hi, I was reading through this discussion and I saw you created a discord group but when I clicked on it, it said the link was invalid.  Is there a way I can join the group?

**Reply by Arsenic-ATG** [reply]
> Hi, I was reading through this discussion and I saw you created a discord group but when I clicked on it, it said the link was invalid. Is there a way I can join the group?

looks like the link has been expired, 
If you have any queries, you may use the [discussion tab](https://github.com/MakeContributions/DSA/discussions) here in the mean time
